static/usage/picker/multiple-column/angular/app_component_ts.md

```diff
@@ -0,0 +1,82 @@
+```ts
+import { Component } from '@angular/core';
+import { PickerController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+})
+export class AppComponent {
+  constructor(private pickerCtrl: PickerController) {}
+
+  async openPicker() {
+    const picker = await this.pickerCtrl.create({
+      columns: [
+        {
+          name: 'meat',
+          options: [
+            {
+              text: 'Pepperoni',
+              value: 'pepperoni',
+            },
+            {
+              text: 'Smoked Ham',
+              value: 'smoked-ham',
+            },
+            {
+              text: 'Crispy Bacon',
+              value: 'bacon',
+            },
+          ],
+        },
+        {
+          name: 'veggies',
+          options: [
+            {
+              text: 'Red onion',
+              value: 'red-onion',
+            },
+            {
+              text: 'Peppers',
+              value: 'peppers',
+            },
+            {
+              text: 'Black olives',
+              value: 'black-olives',
+            },
+          ],
+        },
+        {
+          name: 'crust',
+          options: [
+            {
+              text: 'Pan style',
+              value: 'pan',
+            },
+            {
+              text: 'Hand tossed',
+              value: 'hand-tossed',
+            },
+            {
+              text: 'Stuffed crust',
+              value: 'stuffed-crust',
+            },
+          ],
+        },
+      ],
+      buttons: [
+        {
+          text: 'Cancel',
+          role: 'cancel',
+        },
+        {
+          text: 'Confirm',
+          handler: (value) => {},
```

Comment:
Should there be something in here to illustrate what you can do with the handler?